leaf/extensions/tags.py

```diff
@@ -14,6 +14,11 @@ class TagsCog(commands.GroupCog, name="Tags", group_name="tags"):
     def __init__(self, bot: LeafBot) -> None:
         self.bot = bot
 
+    async def check_perms(self, tag_record,  interaction: discord.Interaction) -> bool:
+        return tag_record["owner_id"] == interaction.user.id \
```

Comment:
Would using paranthesis be cleaner here?
```py
return (
    tag_record["owner_id"] == interaction.user.id
    or ...
    or ...
)
```

Reply:
Done